src/Components/AddTask/AddTask.tsx

```diff
@@ -1,67 +1,114 @@
-import { FormValues, projectType ,IAuth} from './../../Types/Types';
+import { FormValues, projectType, IAuth } from './../../Types/Types';
 
 
 const AddTask: React.FC = () => {
-  const { baseUrl, requestHeaders }:any
-  //Pick<IAuth, 'baseUrl','requestHeaders'> 
-  = useContext(AuthContext);
+  const { baseUrl, requestHeaders }: any
+    //Pick<IAuth, 'baseUrl','requestHeaders'>
+    = useContext(AuthContext);
   const { getToastValue }: void = useContext(ToastContext);
-  const {  projects }: projectType[] = useContext(ProjectContext);
+  // const { projects }: projectType[] = useContext(ProjectContext);
   // console.log(projects, "from addtasks");
   const [isLoading, setIsLoading] = useState(false);
   const navigate = useNavigate();
+  const [userList, setUserList] = useState([])
+  const [projectList,setProjectList]=useState([])
+
 
-  
   const {
     register,
     handleSubmit,
     formState: { errors },
   } = useForm<FormValues>();
 
-  const goBack=()=>{
+  const goBack = () => {
     navigate("/dashboard/tasks");
   }
   const onSubmit: SubmitHandler<FormValues> = async (data: FormValues) => {
     // setIsLoading(true)
     console.log(data);
-    // axios
-    //   .post(`${baseUrl}/Task`, data, { headers: requestHeaders })
-    //   .then((response) => {
-    //     setIsLoading(false)
-    //     getToastValue(
-    //       "success",
-    //       response?.data?.message || "Project added suceessfully"
-    //     );
-    //     navigate("/dashboard/projects");
-    //   })
-    //   .catch((error) => {
-    //     setIsLoading(false)
-    //     getToastValue(
-    //       "error",
-    //       error?.response?.data?.message ||
-    //         "An error occurred. Please try again."
-    //     );
-    //   });
+    axios
+      .post(`${baseUrl}/Task`, data, { headers: requestHeaders })
+      .then((response) => {
+        setIsLoading(false)
+        getToastValue(
+          "success",
+          response?.data?.message || "Project added suceessfully"
+        );
+        navigate("/dashboard/tasks");
+      })
+      .catch((error) => {
+        setIsLoading(false)
+        getToastValue(
+          "error",
+          error?.response?.data?.message ||
+            "An error occurred. Please try again."
+        );
+      });
   }
 
+  // Get users list
+  const getUsersList = () => {
+
+    axios.get(`${baseUrl}/Users/`, {
+      headers: requestHeaders,
+      params: {
+        pageSize: 40,
+
+      }
+
+    })
+      .then((response) => {
+        console.log(response);
+
+        setUserList(response?.data?.data);
+      })
+      .catch((error) => {
+        console.log(error);
+      })
+  }
+  const getAllProjectsList = () => {
+    axios
+      .get(`${baseUrl}/Project/manager`, { headers: requestHeaders })
+      .then((response) => {
+        setProjectList(response?.data?.data);
+      })
+      .catch((error) => {
+        // getToastValue(
```

Comment:
error and toast messages are helpful to users to show them what is going wrong, so we shouldn't remove them completely